viewlet/tests/test_viewlet.py

```diff
@@ -191,6 +200,9 @@ def test_custom_jinja2_environment(self):
         env = get_env()
         self.assertEqual(env.optimized, True)
         self.assertEqual(env.autoescape, False)
+        # Coffin does not support django > 1.7
+        if django.VERSION > (1, 7):
+            return
```

Comment:
Since coffin<2 supports Django unit 1.7, would it make sense to keep support for 1.11 > Django > 1.7 using https://github.com/niwinz/django-jinja (the alternative that coffin recommends) ? Not sure exactly what jinja specific functionality viewlet has, but since 1.11 Django has support for context processors in jinja template, but Django's support could be enough with 1.11